runtime/doc/options.txt

```diff
@@ -2184,6 +2184,26 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option is reset when the 'paste' option is set and restored when
 	the 'paste' option is reset.
 
+				*'filechangenotify'* *'fcnotify'*
+'filechangenotify' 'fcnotify'	string (default "off")
+				local to buffer
+	Watch for changes to a file. How to react for the changes depends on
+	the value of the option. The option can take a list of following values:
+
+		off		Don't react to changes happening to a file.
+				Can't be set with anyother of the following
+				values.
+		autoread	Notify about the changes to a file and prompt
+				the user if there are changes in the current
+				buffer. If there are no changes reload the
+				buffer without prompts.
+				In the absence of this value the user will be
+				prompted for every change.
```

Comment:
how does this interact with the 'autoread' option?

Reply:
if autoread is set, the buffers will be reloaded automatically, else there will be a prompt for reloading the buffer. For now the `autoread` option overrides what is set in here. I will mention it.